docker-for-windows/edge-release-notes.md

```diff
@@ -11,6 +11,17 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.2
+2020-11-24
+
+> [Download](https://desktop.docker.com/win/edge/50063/Docker%20Desktop%20Installer.exe)
+
+### New
+- Use of three digit version number.
+
+### Bug fixes and minor changes
+
+- The WSL 2 shell helper script runs the real docker CLI in WSL 2 when the integration is enabled.
```

Comment:
I don't understand this item. Is there a ticket number?

Reply:
There's none in the public issue tracker, it was caught by the CI.

Reply:
Removing from this changelog, as it was already mentioned before. (My bad!)